### PR TITLE
add Tp-Link WRT3600-v1, Tp-Link 1043ND-v4, Linksys WRT3200ACM-EU

### PR DESCRIPTION
**Pull request by mikysal78**

### CODE REVIEW

**Review by nemesisdesign — COMMENTED** [review]
@mikysal78 looks good, once you have tested these let me know so I can merge.

**Comment by mikysal78** [comment]
test on device 1043nd-v4 .... Device model and image model do not match
I testing upload manual the new firmware and continue with Device model and image model do not match

File: openwisp_firmware_upgrader/hardware.py [hunk 1]
```diff
@@ -20,15 +20,28 @@
                 'ramips-mt76x8-gl-mt300n-v2-squashfs-sysupgrade.bin',
                 {'label': 'GL.iNet GL-MT300N-V2', 'boards': ('GL-MT300N-V2',)},
             ),
+            (
+                'openwrt-ipq806x-generic-tplink_c2600-squashfs-sysupgrade.bin',
```
Comment (nemesisdesign) [inline openwisp_firmware_upgrader/hardware.py:24]: `openwrt` all'inizio della stringa va rimosso

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: openwisp_firmware_upgrader/hardware.py (hunk 1 above)
Comment (nemesisdesign) [reply openwisp_firmware_upgrader/hardware.py:24]: vale anche per le altre occorrenze

**Pull request closed without merging**